azure-toolkit-libs/azure-toolkit-containerapps-lib/src/main/resources/template/aca/artifact-dockerfile

```diff
@@ -1,17 +1,14 @@
 # Dockerfile
 
 # Use a minimal JDK runtime as the base image
```

Comment:
The comment is not very accurate.

I would suggest we delete most of the comments in this file. The instructions are pretty self-explained, we don't need the verbal comments.